cmd/push.go

```diff
@@ -16,10 +16,14 @@
 package cmd
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"net/url"
 	"strings"
+	"time"
+
+	"github.com/nats-io/nats.go"
 
```

Comment:
make fmt - to remove the extra empty lines

Reply:
make fmt does not do that.
I removed them separately